src/Microsoft.Framework.Runtime.Roslyn.Interfaces/IAfterCompileContext.cs

```diff
@@ -21,5 +22,7 @@ public interface IAfterCompileContext
         Stream XmlDocStream { get; set; }
 
         IList<Diagnostic> Diagnostics { get; }
+
+        IEnumerable<IMetadataReference> References { get; }
```

Comment:
Nope, just follow the convention on the current interface for now. We're going to make other changes later.